### DISCUSSION TITLE
Another Vlang Book Has Been Written (4 Published - 2023)

### DISCUSSION THREAD

**Discussion by Wajinn** [vlang/v#20056] [General]
[Randomness Revisited using the V Programming Language](https://novapublishers.com/shop/randomness-revisited-using-the-v-programming-language/), 2023
Note 1: This one is much more scholarly and covers a specific subject, as oppose to the previous ones.
Note 2: See updated book list ([25929](https://github.com/vlang/v/discussions/25929)).

This brings the book total to 4.  The other (known) books on V are:

[The V Programming Language basic](https://www.amazon.com/Programming-Language-basic-Japanese-ebook/dp/B08BKJDRFR) (Japanese), 2020
[Getting Started with V Programming](https://www.amazon.com/Getting-Started-Programming-end-end-ebook/dp/B09FKK3JL7), 2021
[Beginning with V Programming](https://www.amazon.com/Beginning-Programming-beginners-programming-language/dp/B09XSZKTJ1), 2022

**Comment by Delta456** [comment]
It's written by @hungrybluedev

**Comment by hungrybluedev** [comment]
Hi! I'm the author of the book. It has been accepted and is currently at press. It's available for pre-order but I'm still going back and forth with the publishers about the book cover.

So I'm expecting the book to be out by the end of December. 🤞

**Comment by medvednikov** [comment]
Exciting news! Great job.

**Comment by JalonSolov** [comment]
Need to get it added to the Wikipedia page once it's out.  :-)

**Reply by Wajinn** [reply]
Not only that, but more translations into other languages.  Looks like the French and other translations, that were out before the English article, could use updating.

Interestingly and recently, Larry Sanger (co-founder of Wikipedia), [wrote an article](https://larrysanger.org/2023/11/wikipedias-empire-and-how-to-build-a-rebel-alliance-of-knowledge/) about alternatives.  He appears to think highly of [Justapedia](https://justapedia.org/), [Citizendium](https://citizendium.org/), and [Handwiki](https://handwiki.org/).  These are alternatives whose articles show up on [EncycloSearch](https://encyclosearch.org/).

Someone created an article about V on Handwiki, but something appears odd with it, as it doesn't show up through their regular search or EncycloSearch (that pulls from Handwiki).  Not sure what's going on, but it probably should be updated or more people may consider emailing the Handwiki team about that.

**Comment by shove70** [comment]
That's great. 👏 👏 👏
